src/lib/prepareOas.ts

```diff
@@ -69,7 +68,7 @@ export default async function prepareOas(path: string, command: 'openapi' | 'val
   let bundledSpec = '';
 
   if (command === 'openapi') {
-    bundledSpec = await oas.bundle().then((res: Record<string, unknown>) => {
```

Comment:
Removed the type here because `OASNormalize.bundle` returns an `OpenAPI.Document` now and `Record` is incompatible with that.

Reply:
:hell-yeah: